package.json

```diff
@@ -26,7 +27,15 @@
     "url": "git+https://github.com/gnosis/dex-contracts.git"
   },
   "files": [
-    "build"
+    "build/contracts/BatchExchange.json",
+    "build/contracts/EpochTokenLocker.json",
+    "build/contracts/Migrations.json",
+    "build/contracts/SnappAuction.json",
+    "build/contracts/SnappBase.json",
+    "build/contracts/SnappBaseCore.json",
+    "build/contracts/TokenConservation.json",
```

Comment:
Are there any .json files that shouldn't be added? Can we use a wildcard?

Reply:
I think that most of the JSON files shouldn't be added. They include things like `ERC20.json` and `SafeMath.json`. These are all the truffle artifacts for the contracts directly built in this package.